web/src/views/org/OrgRepos.vue

```diff
@@ -6,7 +6,7 @@
 
     <template #titleActions>
       <IconButton
-        v-if="orgPermissions.admin"
+        v-if="!org.is_user && orgPermissions.admin"
```

Comment:
What's the purpose of the changed permission check to the org views?

Reply:
Im a bit confused. AFAIK Woodpecker threads users as orgs. If I look at a repo from my personal user space, this is handled as

Reply:
Thank you for the explanation. Hmm, as you wrote in https://github.com/woodpecker-ci/woodpecker/issues/1739 the backend is the same and as we handle users as orgs now this should IMO just work? Or what's technically missing that there are no user secrets yet? Just removing it because we might want to move user secrets to another location sounds bad to me. But I'm not that deep into the codebase yet either, sorry if the questions are bullshit :see_no_evil:

Reply:
Thanks :+1: works for me as well.